hwilib/base58.py

```diff
@@ -79,7 +79,7 @@ def get_xpub_fingerprint_hex(xpub):
 def get_xpub_fingerprint_as_id(xpub):
     data = decode(xpub)
     fingerprint = data[5:9] + b'\x00' * 16
-    return hexlify(fingerprint[::-1]).decode()
+    return hexlify(fingerprint).decode()[:8]
```

Comment:
I think it is unnecessary to have `[:8]` here since it will always be 8 characters anyways.

Reply:
When I remove it, it adds lots of zeros at the end.

Reply:
Remove `+ b'\x00' * 16` from the line above

Reply:
Done